exsrv.cmd

```diff
@@ -0,0 +1,71 @@
+:: Utility Batch File To Start/Stop Elixir App As Windows Service
+:: Onorio Catenacci
+:: 19 November 2015
+
+@echo off
+SET THISFILENAME=%~n0
+SET COMMAND=%2
+SET ERTS_BIN_PATH=C:/progra~1/erl7.1/erts-7.1/bin
+SET ERLSRV_CMD=%ERTS_BIN_PATH%/erlsrv
+SET APPNAME=%1
+
+IF [%1]==[] (
+CALL :HELP %THISFILENAME%
+GOTO :EOF
+)
+IF /I %COMMAND%==INSTALL (CALL :INSTALL)
+IF /I %COMMAND%==START (CALL :START)
+IF /I %COMMAND%==STOP (CALL :STOP)
+IF /I %COMMAND%==REMOVE (CALL :REMOVE)
+IF /I %COMMAND%==LIST (CALL :LIST)
+GOTO :EOF
+
+:HELP
+SETLOCAL
+ECHO %~1 appname command
+ECHO Where command can be "INSTALL", "START", "STOP", "LIST" or "REMOVE"
+ECHO Note that installing the service doesn't automatically start it but
+ECHO removing a service will attempt to stop it before removing it.
+ECHO You do have to pass an appname for the list command although the command
+ECHO will list all erlsrv apps regardless.
+ENDLOCAL
+GOTO :EOF
+
+:INSTALL
+SETLOCAL
+SET ELIXIR_HOME=/programdata/chocolatey/lib/Elixir
```

Comment:
exsrv.cmd was a variant cmd file that I was working on.  It never got
merged and in fact I had forgotten I'd even submitted a pull request for
it.

It's not part of the real EXRM package and it never was.  As I say it was a
variant I was working on--which I later discovered I didn't need.  I'll
cancel the PR.

--
Onorio


On Tue, Mar 15, 2016 at 1:39 PM, Steven Critchfield <
notifications@github.com> wrote:

> In exsrv.cmd
> <https://github.com/bitwalker/exrm-test/pull/5#discussion_r56206914>:
>
> > +GOTO :EOF
> > +
> > +:HELP
> > +SETLOCAL
> > +ECHO %~1 appname command
> > +ECHO Where command can be "INSTALL", "START", "STOP", "LIST" or "REMOVE"
> > +ECHO Note that installing the service doesn't automatically start it but
> > +ECHO removing a service will attempt to stop it before removing it.
> > +ECHO You do have to pass an appname for the list command although the command
> > +ECHO will list all erlsrv apps regardless.
> > +ENDLOCAL
> > +GOTO :EOF
> > +
> > +:INSTALL
> > +SETLOCAL
> > +SET ELIXIR_HOME=/programdata/chocolatey/lib/Elixir
>
> Same basic issue with this as previous note, exrm should be packaging
> everything together and therefore shouldn't need a chocolatey directory.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/bitwalker/exrm-test/pull/5/files#r56206914>
>



-- 
Onorio Catenacci

http://onor.io
http://www.google.com/+OnorioCatenacci